.editorconfig

```diff
@@ -22,3 +22,6 @@ ij_continuation_indent_size = 4
 # Doc: https://youtrack.jetbrains.com/issue/IDEA-170643#focus=streamItem-27-3708697.0-0
 # $ means "static"
 ij_java_imports_layout = $*,|,*
+
+[*.json]
+insert_final_newline = false
```

Comment:
Can we rather add a final newline in the committed files?
Otherwise Git (and GitHub UI) emit warnings on the missing newlines :-/

Reply:
Sadly no. This is the way that TUF outputs them and the signing and hash checks depend on them. :( Is there a way  to scope down the editor config to a path?

Reply:
You can put file path into the square brackets: https://editorconfig.org/#file-format-details

For instance, you can put `[test/resources/dev/sigstore/tuf/**/*.json]`, or you can add a new `.editorconfig` file (e.g. inside `test/resources/dev/sigstore/tuf` directory).

Reply:
In any case, please add a comment above that `insert_final_newline` so the others see the reason.

Reply:
done